cmd/dcos/main.go

```diff
@@ -18,7 +18,7 @@ func main() {
 	}
 }
 
-func run(ctx api.Context, args []string) error {
+func run(ctx api.Context, _ []string) error {
```

Comment:
Why not remove this parameter completely when we're not using it?

Reply:
Fixed